src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -75,19 +77,17 @@ export default class Sen extends Module {
 		this.sen[sen] = true
 	}
 
-	private remove() {
-		this.sen = _.mapValues(this.sen, () => false)
-	}
+	private overwrite() {
+
+		this.wasted = this.wasted - 1
 
-	private onHagakure() {
-		// Work out how many sen are currently active
-		const activeSen = Object.values(this.sen)
-			.filter(active => active)
-			.length
+		if (this.wasted < 0) {
+			this.wasted = 0
+			}
```

Comment:
Might just be github but the indenting here looks jank. If it's actually the case, it merits fixing.